keylime-agent.conf

```diff
@@ -11,7 +11,17 @@
 # The configuration file version
 #
 # To override, set KEYLIME_AGENT_VERSION environment variable.
-version = "2.2"
+version = "2.4"
```

Comment:
What happened to 2.3 version? Is this 2.2 -> 2.4 jump expected?

Reply:
Look at the first commit in the series. 2.3 takes into account some idevid options.

Reply:
as @sergio-correia commented, the first commit in the series is the bump to the `2.3` version to account for the IDevID/IAK related configuration options.

I recommend reviewing the commits one by one instead of the whole diff from master as it makes it easier to follow the changes.